Dockerfile.fuzz

```
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine as builder
+
+RUN apk add --no-cache python3 py3-pip bash coreutils
+
+RUN wget -q "https://github.com/microsoft/restler-fuzzer/archive/refs/tags/v9.2.2.tar.gz"
+RUN tar -xvf v9.2.2.tar.gz && mv restler-fuzzer-9.2.2 restler-fuzzer
+
+RUN cd restler-fuzzer; mkdir -p restler_bin
```

```suggestion
RUN mkdir -p /restler-fuzzer/restler_bin
```

Can combine into the previous RUN command as well.